src/components/Info/styled.js

```diff
@@ -0,0 +1,52 @@
+import styled from "@emotion/styled";
+
+export const InfoContainerCustom = styled.div`
+  display: flex;
+  position: fixed;
+  top: 10%;
+  left: 70%;
+  border-radius: 20px;
+  width: 290px;
```

Comment:
For EVERY width, height, margin, padding, please use rem value.
To convert pixels to rem, you have to divide pixels number by 16.
eg: 290 / 16 = 11.8 so you can replace 290px by 12rem instead.